app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    buildToolsVersion '25.0.0'
```

Comment:
If you change this, then please update to latest version 25.0.3 (https://developer.android.com/studio/releases/build-tools.html)